File: Desktop/ailydian-ultra-pro/public/docs/sdks/java/src/main/java/com/lydian/sdk/InsanIQClient.java
package com.lydian.sdk;

import com.lydian.sdk.models.Persona;
import com.lydian.sdk.models.Skill;
import com.lydian.sdk.models.ChatSession;

import java.io.IOException;

/**
 * Client for İnsan IQ API
 */
public class InsanIQClient {
    private final LydianClient client;

    public InsanIQClient(LydianClient client) {
        this.client = client;
    }

    public Persona createPersona(Persona persona) throws IOException {
        return client.request("POST", "/insan-iq/personas", persona, Persona.class);
    }

    public Skill publishSkill(Skill skill) throws IOException {
        return client.request("POST", "/insan-iq/skills", skill, Skill.class);
    }

    public ChatSession createSession(ChatSession session) throws IOException {
        return client.request("POST", "/insan-iq/sessions", session, ChatSession.class);
    }
}

File: public/docs/sdks/csharp/InsanIQClient.cs
using System.Threading;
using System.Threading.Tasks;
using Lydian.SDK.Models;

namespace Lydian.SDK
{
    public class InsanIQClient
    {
        private readonly LydianClient _client;

        public InsanIQClient(LydianClient client)
        {
            _client = client;
        }

        public async Task<Persona> CreatePersonaAsync(Persona persona, CancellationToken cancellationToken = default)
        {
            return await _client.RequestAsync<Persona>("POST", "/insan-iq/personas", persona, cancellationToken)
                ?? throw new LydianException("Failed to create persona", 500);
        }

        public async Task<Skill> PublishSkillAsync(Skill skill, CancellationToken cancellationToken = default)
        {
            return await _client.RequestAsync<Skill>("POST", "/insan-iq/skills", skill, cancellationToken)
                ?? throw new LydianException("Failed to publish skill", 500);
        }

        public async Task<ChatSession> CreateSessionAsync(ChatSession session, CancellationToken cancellationToken = default)
        {
            return await _client.RequestAsync<ChatSession>("POST", "/insan-iq/sessions", session, cancellationToken)
                ?? throw new LydianException("Failed to create session", 500);
        }
    }
}
